data/avro-schema/src/main/resources/avro/net/corda/data/permissions/management/user/ChangeUserParentGroupIdRequest.avsc

```diff
@@ -4,7 +4,7 @@
   "namespace": "net.corda.data.permissions.management.user",
   "fields": [
     {
-      "name": "userId",
+      "name": "loginName",
```

Comment:
In theory, this is a breaking change - so we might be better off incrementing version of `corda-api` in this PR.